src/test/java/org/example/ProductsTests.java

```diff
@@ -0,0 +1,35 @@
+package org.example;
+
+import Pages.ProductPage;
+import org.assertj.core.api.Assertions;
+import org.example.baseTests.BaseTests;
+import org.testng.annotations.Test;
+
+public class ProductsTests extends BaseTests {
+
```

Comment:
add here `private static final String productToBeSearched = "Apple iPhone 11 Pro 64"; ` and use this variable in tests instead of regular text